app/views/organisations/_contacts.html.erb

```diff
@@ -21,7 +21,7 @@
         <div class="<%= contact[:post_addresses].any? ? "govuk-grid-column-one-half" : "govuk-grid-column-two-thirds" %>">
           <% if contact[:email_addresses].any? %>
             <div class="organisation__margin-bottom">
-              <p <%= "lang=en" if contact[:locale] != "en" %>><%= t('organisations.contact.email', locale: :en) %></p>
+              <p <%= "lang=en" if contact[:locale] != "en" %>><%= t('organisations.contact.email') %></p>
```

Comment:
When we are viewing a translated version of a page, and a translation in that language exists for a particular string/sentence/term on the page, then that string should not have a `lang="en"` attribute. 

On the [Welsh version of this page](https://govuk-collec-display-co-i94w6w.herokuapp.com/government/organisations/land-registry.cy), the term "E-bost" is now being indicated as being in English which is not accurate:
<img width="1677" alt="Screenshot 2022-01-14 at 12 21 20" src="https://user-images.githubusercontent.com/7116819/149514776-e89a7af0-b585-43eb-a715-a026e6935e7d.png">
 

If, however, a translation does not exist for an element, and i18n  is falling back to the English language default, then that element should have `lang="en"`. So let's say a translation did not exist for `t('organisations.contact.email')` so it had to fall back to the English language term "Email" – that is when a `lang` attribute is required to correctly indicate the language that bit of text is in.

Reference: https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html

Reply:
So in this case, the a Welsh language translation does exist, and all of the addresses are in Welsh, and the "div" it's contained in is also Welsh:

<img width="1536" alt="Screenshot 2022-01-14 at 13 25 15" src="https://user-images.githubusercontent.com/5793815/149522434-4f7b7160-2362-4ea1-a4cb-bd52335659e6.png">

Should I remove the `if` statement on this line as well: https://github.com/alphagov/collections/blob/main/app/views/organisations/_contacts.html.erb#L24 ?

Reply:
Perhaps it would be better to check if the translation for the language exists before replacing it? https://stackoverflow.com/questions/12353416/rails-i18n-check-if-translation-exists